### PR TITLE
[docs-only] wait for collabora to start before trying to register app

### PR DESCRIPTION
**Pull request by individual-it**
## Description
the apps were not always registered in the UI for me, restarting ocis-appprovider-collabora fixes the problem, so I've added a health-check and a depends condition, to make it reliable

Test together with #8143 because there are other issues

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: deployments/examples/ocis_wopi/docker-compose.yml [hunk 1]
```diff
@@ -127,6 +127,9 @@ services:
     logging:
       driver: "local"
     restart: always
+    depends_on:
+      collabora:
+        condition: service_healthy
```
Comment (wkloucek) [inline deployments/examples/ocis_wopi/docker-compose.yml:132]: This is only part of V2, right? V3 https://docs.docker.com/compose/compose-file/compose-file-v3/#depends_on doesn't mention it.

File: deployments/examples/ocis_wopi/docker-compose.yml [hunk 2]
```diff
@@ -104,7 +104,9 @@ services:
     image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
     networks:
       ocis-net:
-    command: app-provider server
+    entrypoint:
+      - /bin/sh
+    command: ["-c", "curl -k --fail https://${COLLABORA_DOMAIN:-collabora.owncloud.test} || exit 1 && ocis app-provider server" ]
```
Comment (wkloucek) [inline deployments/examples/ocis_wopi/docker-compose.yml:109]: looks quite similar to what we already have inhttps://github.com/owncloud/ocis/blob/master/deployments/examples/ocis_wopi/config/ocis-appprovider-onlyoffice/entrypoint-override.sh

Could that be reused?

Comment (individual-it) [reply deployments/examples/ocis_wopi/docker-compose.yml]: done

File: deployments/examples/ocis_wopi/docker-compose.yml [hunk 3]
```diff
@@ -209,6 +214,13 @@ services:
     logging:
       driver: "local"
     restart: always
+    healthcheck:
```
Comment (2403905) [inline deployments/examples/ocis_wopi/docker-compose.yml:217]: Does it make sense to use the `healthcheck` at all? Does `depends_on` respect `healthcheck`?
If it works, we also should add it for the onlyoffice.

**Comment by DeepDiver1975** [comment]
I think this can be closed.

**Pull request closed without merging**